gulp/tasks/build.js

```diff
@@ -172,13 +166,13 @@ function compileIndvES2015ModulesToBrowser() {
         },
         footer: fileName => {
           return isFirebaseApp(fileName) ? `
-          })();` : `
+          })().default;` : `
```

Comment:
We should probably leave a comment here pointing to https://github.com/babel/babel/issues/2212 for future reference as to why this is needed.